runtime/src/accounts.rs

```diff
@@ -87,7 +87,7 @@ impl AccountLocks {
 }
 
 /// This structure handles synchronization for db
-#[derive(Default, Debug, AbiExample)]
```

Comment:
Does the default impl for Default not work here?

Reply:
derive impl Default won't once AccountsDb::default() disappears, which is coming soon.